docs/oauth2.md

```diff
@@ -62,149 +71,79 @@ auth_backends.1 = rabbit_auth_backend_oauth2
 
 Next, let's take a look at the workflows the OAuth 2 plugin supports.
 
-### Authorization Workflow {#authorization-workflow}
-
-This plugin does not communicate with any OAuth 2.0 provider. It decodes an access token provided by the client and authorises a user based on the data stored in the token.
-
-The token can be any [JWT token](https://jwt.io/introduction/) which contains the `scope` and `aud` fields. The way the token was issued (such as what grant type was used) is outside of the scope of this plugin.
-
 ### Prerequisites {#prerequisites}
 
 To use this plugin, all RabbitMQ nodes must be
 
 1. [configured to use the rabbit_auth_backend_oauth2 backend](./access-control).
```

Comment:
"configured" should have a capital "C" it is the start of a sentence same for 2. and 3. below